test/functional/Java8andUp/src_90/org/openj9/test/java/lang/Test_String_ToLowerCase_Lithuanian.java

```diff
@@ -0,0 +1,144 @@
+package org.openj9.test.java.lang;
+
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeMethod;
+import org.openj9.test.support.Support_Exec;
+import org.testng.Assert;
+import org.testng.AssertJUnit;
+import java.io.BufferedWriter;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.io.Writer;
+import java.security.Permission;
+import java.util.Enumeration;
+import java.util.Locale;
+import java.util.Properties;
+
+@Test(groups = { "level.sanity" })
+public class Test_String_ToLowerCase_Lithuanian {
+	/**
+	 * @tests java.lang.String#toLowerCase(java.util.Locale) using Unicode 8.0.0 standard which is supported in Java 9
+	 */
+	@Test
+	public void test_toLowerCase_lithuanian() {
```

Comment:
The test is the same between Java 8 and later. Can we have a different helper that populates the accents array but share the test code?

Reply:
Or just use the same helper but test the java version at runtime.

Reply:
yes, I may have directed you without knowing enough about the difference between versions...  In this case, since the API is not different, seems you could in fact just check what version inside the helper (https://github.com/eclipse/openj9/blob/master/test/functional/TestUtilities/src/org/openj9/test/util/VersionCheck.java).

Reply:
I agree. I've consolidated the tests. Fixed in 0bee8a2